tests/golden/check-color/expected-color.gold

```diff
@@ -0,0 +1,12 @@
+=== Repository data ===
+
+  [96mcolor.md[0m:
+    - references:
+        - reference ([92mfile-local[0m) [2mat src:9:1-15[0m:
+            - text: "Color"
+            - link: -
+            - anchor: Color
+    - anchors:
+        - color ([2m[92mheader I[0m[0m) [2mat src:7:1-7[0m
+
+All repository links are valid.
```

Comment:
Yeah, I think this format is just the best ))

Reply:
Yeah :satisfied:, as you may have seen in the commit history, I was trying to store this in a textual representation of the binary data, but encountered a lot of problems, mainly:

* Different behaviour, parameters and output on macOS and linux commands.
* Different Windows newline character.

Reply:
Ugh, that sounds like a sufferring :yum: